client/securedrop_client/storage.py

```diff
@@ -505,6 +511,9 @@ def __update_submissions(
                 f"Tried to delete submission {deleted_submission.uuid}, but "
                 "it was already deleted locally."
             )
+        except SQLAlchemyError as e:
```

Comment:
I don't follow what this is for, there's already a clause for NoResultFound?

Reply:
There are other error SQLAlchemy error types that could occur (eg InvalidRequest) after the delete, but trying to refresh my memory here I think I was waffling a bit on whether to include generic SQLAlchemy error handling in this PR or not. (And I probably would need to include the commits too if I did).  I'm going to take another look but probably roll it back for now. ty!

Reply:
Removed in https://github.com/freedomofpress/securedrop-client/commit/138643be480dd82bb9ed24b787a04307a8d8c1ae